debian/marquee/00-nginx-conf

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+apt install nginx
+
+rm -f /etc/nginx/sites-enabled/*
+
+cp nginx/conf/http.conf /etc/nginx/conf.d/
+cp nginx/conf/whatwg.conf /etc/nginx/
+cp nginx/conf/whatwg-headers.conf /etc/nginx/
+
+chown -R deploy:deploy /var/www
+rm -rf /var/www/html
+
+mkdir -p /var/www/http/.well-known/acme-challenge
```

Comment:
I was going to say put this before the `chown`, but maybe it's just as well that it's after so that only root can create files there? On marquee it's currently owned by the deploy user, so one needs to change.

Reply:
> I was going to say put this before the `chown`, but maybe it's just as well that it's after so that only root can create files there? On marquee it's currently owned by the deploy user, so one needs to change.

I’ve moved it before the `chown`. It doesn’t seem like having only root be able to create files there needs to be a big concern